crates/store/re_grpc_server/src/lib.rs

```diff
@@ -298,8 +298,16 @@ pub fn spawn_with_recv(
                 }
             };
             match msg {
-                Ok(msg) => {
-                    if channel_log_tx.send(msg).is_err() {
+                Ok(mut log_msg) => {
+                    // Insert the timestamp metadata into the Arrow message for accurate e2e latency measurements.
+                    // Note that this function is only called by the viewer
```

Comment:
tbh I'm too confused about our stack to be confident on this statement, but I trust you thought long and hard about it 😅